### PR TITLE
EventHub: added optional keyVersion property in KeyVaultProperties

### PR DESCRIPTION
**Pull request by v-Ajnava**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [X] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [X] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [X] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/namespaces-preview.json [hunk 1]
```diff
@@ -520,6 +520,10 @@
         "keyVaultUri": {
           "type": "string",
           "description": "Uri of KeyVault"
+        },
+        "keyVersion": {
```
Comment (majastrz) [inline specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/namespaces-preview.json:524]: >keyVersion [](start = 9, length = 10)

Discussed over IM: The property was missed from original swagger.

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Comment by allenjzhang** [comment]
Discussed offline, per submitter, this API-Version have not been release and **the way it stands now would not work without the fix anyway**. Although looks like Python has included this API version and CLI is using Python SDK. 

I have pinged CLI folks (@yungezz ) to confirm impact if any and coordinating update. Keeping this PR open for now. Thanks for your patience.

**Comment by v-Ajnava** [comment]
@yungezz , we are not using 2018-01-01-preview API version in the CLI. currently only 2017-04-01 stable API version is used in the CLI for EventHub

**Comment by yungezz** [comment]
hi @v-Ajnava I found 2018-01-01preview api version is in Python SDK https://pypi.org/project/azure-mgmt-eventhub/3.0.0/#files. Azure CLI didn't use it, but customer might use it. I didn't check other languages SDKs.

**Comment by v-Ajnava** [comment]
@yungezz, without the fix in the current PR the BYOK functionality would not work and is broken in the existing 2018-01-01-preview.

Previously the swagger was accepting single object of KeyVaultProperties which is broken, services accepts array of KeyVaultProperties. 

please let me know your views.

**Pull request merged**